sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -2212,6 +2255,18 @@ public partial class ManagedTransparentDataEncryptionResource : Azure.ResourceMa
         public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Sql.ManagedTransparentDataEncryptionResource> Update(Azure.WaitUntil waitUntil, Azure.ResourceManager.Sql.ManagedTransparentDataEncryptionData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Sql.ManagedTransparentDataEncryptionResource>> UpdateAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.Sql.ManagedTransparentDataEncryptionData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct MinimalTlsVersion
```

Comment:
this name is quite generic, could we rename it to be more specific? We could have `SqlMinimalTlsVersion` or you could add other things as the context.

Reply:
Please note that since we have a customization code for this model already, after you change the name of this by using `rename-mapping`, we should also update the type names in your customization code to align the new name.

Reply:
Earlier also it was MinimalTlsVersion so in that sense minTlsVersion should be ok and more similar to the original name I feel.

Reply:
We would like type names to be more consistent between libraries as well, usually this is called XXXMinimalTlsVersion which is the reason I suggest `SqlMinimalTlsVersion`. The name `MinTlsVersion` is an unfortunate workaround.